src/js/component/TeamMain.js

```diff
@@ -0,0 +1,43 @@
+import Component from "../libs/component.js";
+import { $ } from "../utils/dom.js";
+import { SELECTORS, POPUP_MESSAGES } from "../utils/constant.js";
+import { isAvailableTeamName } from "../utils/validators.js";
+import { ERROR_HANDLER } from "../utils/errors.js";
+import { getState, dispatch } from "../redux/functions.js";
+import { teamListView, loadingView } from "../utils/templates.js";
+import { ACTIONS } from "../actions/team/index.js";
+
+class TeamMain extends Component {
+  constructor() {
+    super();
+    this.container = $(SELECTORS.TEAM_LIST);
+    this.bindEvents();
+    this.render();
+  }
+
+  bindEvents() {
+    const $addTeamButton = $(SELECTORS.TEAM_ADD_BTN);
+    $addTeamButton.addEventListener("click", (e) => this.onClickAddButton(e));
```

Comment:
```suggestion
$addTeamButton.addEventListener("click", this.onClickAddButton);
```
이렇게만 작성해주셔도 Event 객체는 자동으로 넘어갑니다 😄

Reply:
아하! 감사합니다 😄